composer.json

```diff
@@ -10,8 +10,8 @@
   ],
   "require": {
     "php": "^5.2|^7|^8",
-    "npm-asset/chartist": "^0.11.4",
-    "npm-asset/chartist-plugin-tooltips-updated": "^0.1.4"
```

Comment:
Why the removal of the `^` here and in the previous line?

Reply:
We previously had version `^0.11.4` and a _composer.lock_ file that locked the version to `0.11.4` when installing. We now remove the lockfile for convenience (we otherwise would have to keep at least 3 locks for different PHP generations, so Composer would pick up any Version `0.11.4 <= v < 1`, so we lock it this way.

Doesn't make any difference for this specific dependency, because 0.11.4 is effectively the latest version below 1 and the Update is already in place (#239).

Reply:
ah okay, thanks!